README.md

```diff
@@ -131,14 +137,17 @@ export.status
 
 ### Panels
 
-To get all the panels:
+To paginate all the panels:
 
 ```ruby
-QualtricsAPI.panels.fetch
+panels = QualtricsAPI.panels.fetch
+# => #<QualtricsAPI::PanelCollection:0x007f8769aae2c0 ....>
+
+panels.fetch # get next page
```

Comment:
this is a bit odd... let's talk about it a little bit more when i go back. 
I think the functionality of `fetch` should be consistent (always do the same thing - return the same results - and has no side effects) 
pagination should be implemented in separate methods.